src/scene/sprite-nine-slice/NineSliceSpritePipe.ts

```diff
@@ -78,6 +78,7 @@ export class NineSliceSpritePipe implements RenderPipe<NineSliceSprite>
 
     private _updateBatchableSprite(sprite: NineSliceSprite, batchableSprite: BatchableMesh)
     {
+        sprite.didViewUpdate = false;
```

Comment:
does it work without this line @norflin321 ?

Reply:
@GoodBoyDigital yes